pkg/cmd/dcos_test.go

```diff
@@ -33,6 +33,8 @@ Commands:
       Manage the DC/OS configuration file
   help
       Help about any command
+  plugin
+      Manage CLI plugins
```

Comment:
Should this be in a separate PR/commit?

Reply:
This was present in different PRs, as I've merged the `dcos plugin remove` PR it is not in this one anymore.